packages/eslint-config-airbnb-base/rules/best-practices.js

```diff
@@ -164,9 +164,9 @@ module.exports = {
     'no-octal-escape': 'error',
 
     // disallow reassignment of function parameters
-    // disallow parameter object manipulation
+    // disallow parameter object manipulation except req, request, and $scope
     // rule: http://eslint.org/docs/rules/no-param-reassign.html
-    'no-param-reassign': ['error', { props: true }],
+    'no-param-reassign': ['error', { props: true, ignorePropertyModificationsFor: ['req', 'request', '$scope'] }],
```

Comment:
also let's split the options object (and the array) onto multiple lines, so that each exclusion can get a comment describing why it's in the array.